docs/source/transforms.rst

```diff
@@ -51,12 +51,37 @@ Make sure to use only scriptable transformations, i.e. that work with ``torch.Te
 
 For any custom transformations to be used with ``torch.jit.script``, they should be derived from ``torch.nn.Module``.
 
+In TorchVision we implemented 3 policies learned on the following datasets: ImageNet, CIFA10 and SVHN.
```

Comment:
This phrase looks a bit out of its place? I think It should be together with Autoaugment instead of the scriptable transforms section